docs/source/contributing/style-guide.md

```diff
@@ -11,9 +11,9 @@ myst:
 
 ## About design systems
 
-A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product. So a Design System is not a deliverable, but a set of deliverables. It will evolve constantly with the product, the tools and the new technologies.
+A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product. So a Design System is not a deliverable, but a set of deliverables. It constantly evolves with the product,tools and new technologies.
```

Comment:
Our style guides uses one sentence per line, although older docs such as this file did not follow this practice.

```suggestion
A Design System is the single source of truth which groups all the elements that will allow the teams to design, realize and develop a product.
So a Design System is not a deliverable, but a set of deliverables.
It constantly evolves with the product, tools, and new technologies.
```